glib/src/param_spec.rs

```diff
@@ -1370,6 +1370,77 @@ impl ParamSpecVariant {
     }
 }
 
+pub trait Parametrize {
+    type Spec;
+    fn get(&self) -> crate::Value;
+}
+pub trait ParametrizeMut {
+    fn set(&self, value: &crate::Value);
+}
+
+use crate::ToValue;
+impl Parametrize for String {
+    type Spec = ParamSpecString;
+    fn get(&self) -> crate::Value {
+        self.to_value()
+    }
+}
+impl Parametrize for crate::Object {
```

Comment:
This should be moved into the `glib::wrapper!` macro

Reply:
I'm currently writing some bad, dirty code, because it's my first proc macro. The first commits are going to be messy.
Then I'll move everything to the right place.